src/languageserver/handlers/languageHandlers.ts

```diff
@@ -207,6 +211,15 @@ export class LanguageHandlers {
     return this.languageService.getFoldingRanges(textDocument, context);
   }
 
+  selectionRangeHandler(params: SelectionRangeParams): SelectionRange[] {
```

Comment:
I think I missed the typing since tsconfig strict is not set to true, and just taking a look, a lot of other functions are missing undefined return type as well, should we fix them too?